py/setup.py

```diff
@@ -0,0 +1,32 @@
+import os
+import setuptools
+import shutil
+
+# Copies the schema in the module so that setuptools is able to
+# find the file and add it to the package.
+if os.path.isfile("../relay/event.schema.json"):
+    shutil.copyfile(
+        "../relay/event.schema.json",
+        "./sentry_data_schemas/event.schema.json"
+    )
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="sentry-data-schemas",
+    version="0.0.1",
+    author="Sentry",
+    license="BSL-1.1",
+    author_email="hello@sentry.io",
```

Comment:
I htink we have `oss@sentry.io` @BYK ?

Reply:
Yup, thanks for the heads up @mattrobenolt!

```suggestion
    author_email="oss@sentry.io",
```